analytics/src/main/java/com/segment/analytics/AnalyticsContext.java

```diff
@@ -139,6 +142,7 @@ static synchronized AnalyticsContext create(
         analyticsContext.putNetwork(context);
         analyticsContext.putOs();
         analyticsContext.putScreen(context);
+        analyticsContext.putTelemetry();
```

Comment:
Rename to `putInstanceId()` ??

Reply:
yah. I thought about this too. but seems like the "put" functions are grouped by functionality (network, os, app, library, etc). the `instanceId` is added specifically for telemetry. we might have more fields to support telemetry in the future. thus, thought it's better to put it into a new group